src/core/interfaces.ts

```diff
@@ -476,6 +476,29 @@ export namespace CoreChartProps {
   export type I18nStrings = CartesianI18nStrings & PieI18nStrings;
 }
 
+export interface CoreLegendProps {
+  series: Highcharts.SeriesOptionsType[];
+  title?: string;
+  actions?: React.ReactNode;
+  position?: "bottom" | "side";
```

Comment:
This only makes sense when using within a chart. For standalone legends we can rename this to e.g. `alignment: "horizontal" | "vertical"`.